docs/content/v2.6/secure/authorization/rbac-model.md

```diff
@@ -44,6 +44,64 @@ Roles in YSQL can represent individual users or a group of users. They encapsula
 
 * Roles inherit the privileges of all other roles granted to them.
 
```

Comment:
Same comment as /stable... You may need to do two separate installations (2.8.0 and 2.6.7) with three separate data directories just to make extra-sure that the role lists are right, btw.

https://download.yugabyte.com/ is the easiest way to grab those all in one spot.